nifi-nar-bundles/nifi-gcp-bundle/nifi-gcp-processors/src/test/java/org/apache/nifi/processors/gcp/bigquery/PutBigQueryBatchIT.java

```diff
@@ -32,6 +32,10 @@
 import java.nio.file.Files;
 import java.nio.file.Path;
 
+/**
+ * @deprecated {@link PutBigQueryIT} is covering the needed validation for the new class.
+ */
+@Deprecated
```

Comment:
Deprecating the integration test is generally unnecessary. It would actually be worth removing the integration test, because it is not run as part of standard builds.

Reply:
Wasn't sure what to do here. I agree with you I removed the test.